addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -513,7 +505,10 @@ registerModel({
                 }
             };
             this._peerConnections[token] = peerConnection;
-            this._dataChannels[token] = dataChannel;
+            this.messaging.models['RtcDataChannels'].insert({
+                dataChannel,
```

Comment:
This is done in reverse, the RtcDataChannels model should create the actual channel, not the other way around.
Ok to do in another PR if that allows reducing diff here.

Reply:
Also model name should be singular

Reply:
Let's do this in another PR. Keep focus on just turning props in fields with this PR and others.
At a glance, create `dataChannel` in `RtcDataChannel` is more refactoring. `_createPeerConnection` is quite big and lots of call to other methods in `Rtc`.
And also `_peerConnections` has the same issues, so this would lead to big PR if we turn into fields and make this change.